src/index.html

```diff
@@ -1,12 +1,400 @@
+            >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#phone&pc"
+              class="nav__link"
+            >Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#notebook&pc"
+              class="nav__link"
+              data-qa="nav-hover"
+            >Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#gadjet"
+              class="nav__link"
+            >Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#tablet"
+              class="nav__link"
+            >Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >Фото</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="container__cards">
+        <div class="card container__card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="mac photo"
+          >
+
+          <div class="card__title">
+            <ul class="card__title-text">
+              <li>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</li>
+              <li class="card__title-text_small">Код товара: 195434</li>
+            </ul>
+          </div>
+
+
+          <div class="card__feetback">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__responce">
+              <p>Отзывов: 5</p>
+            </div>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Цена:</p>
+            <p class="card__price-coast">69 999 грн</p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >Купить</a>
+        </div>
+
+        <div class="card container__card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="mac photo"
+          >
+
+          <div class="card__title">
+            <ul class="card__title-text">
+              <li>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</li>
+              <li class="card__title-text_small">Код товара: 195434</li>
+            </ul>
+          </div>
```

Comment:
Don't need to create a list here
```suggestion
          <div class="card__title">
            <h2>
              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
            </h2>
            <span>
              Код товара: 195434
            </span>
          </div>
```